docs/bootstrap-design.md

```
@@ -0,0 +1,90 @@
+```python
+# Poll interval: 0.1s (not 1s) for responsive startup
+while waited < max_wait:
+    if ctrl.status == "IDLE":
+        break
+    await asyncio.sleep(0.1)  # Fast polling
+    waited += 0.1
+```
+
+### Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)